src/flask_paranoid/paranoid.py

```diff
@@ -33,6 +36,8 @@ def before_request():
                     response = redirect(url)
                 self.clear_session(response)
```

Comment:
Shouldn't the clear session call be removed, at least when the `many` option is set?